app/models/deck.rb

```diff
@@ -0,0 +1,10 @@
+class Deck < ActiveRecord::Base
+  belongs_to :user
+  has_many :cards
+
+  scope :get_current, ->{ where('current = true') }
```

Comment:
Space missing to the left of {.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.